### DISCUSSION TITLE
How to transfer/migrate a project to another AWS account (Hubs Cloud)

### DISCUSSION THREAD

**Discussion by AudunSpello** [mozilla/hubs#5702] [Hubs Cloud Questions] [answered]
I am developing a Hubs Cloud project for a client. We have set it up in our AWS account. Do I have to manually recreate the project on the client's AWS account, or is there a way to easily transfer/migrate the AWS project to the client's account?

**Comment by pattersonbl2** [comment, marked as answer]
These two bits of information may help with this. 

[Backup Project](https://hubs.mozilla.com/docs/hubs-cloud-aws-backup-and-restore.html)
[Possible Migration of Project](https://aws.amazon.com/premiumsupport/knowledge-center/rds-snapshots-share-account/.)


You would have to create an RDS snapshot of the project, then share it across accounts.  We have yet to try this, so this could cause issues with the project in the second account.
